packages/site/src/routes/events/[slug]/+page.svelte

```diff
@@ -4,17 +4,63 @@
   import Seo from "$lib/components/Seo.svelte";
   import Image from "$lib/components/Image.svelte";
   import { PortableText } from "@portabletext/svelte";
+  import { atcb_action } from 'add-to-calendar-button';
+  import blocksToHtml from '@sanity/block-content-to-html'
 
   import type { PageData } from "./$types";
 
   export let data: PageData;
 
   let event = data.event;
-  $: event = data.event;
-
+  
   function formatDate(date: string) {
     return format(new Date(date), "M/dd/yyyy 'at' h:mm a");
   }
+
+  const toHtml = (blocks: PortableText): HTMLElement => {
+    return blocksToHtml({blocks})
+  }
+
+  const toHtmlString = (blocks: PortableText): string => {
+    return toHtml(blocks).innerHTML;
+  }
```

Comment:
Can you move this part to the server? That way we're not shipping down `@sanity/block-content-to-html` and the Sanity portable text Svelte thing too.

Reply:
Agree, I replaced the description with something generic. Probably better to limit the scope of this and then enhance it later in server code?

Reply:
Makes sense!